doc/articles/features/using-linux-framebuffer.md

```diff
@@ -24,15 +24,15 @@ dotnet new unoapp -o MyApp
 
 You'll get a set of projects, including one named `MyApp.Skia.Linux.FrameBuffer`.
 
-You can build this app by navigating to the `MyApp.Skia.Linux.FrameBuffer` and type the following:
+You can build and run this app by navigating to the `MyApp.Skia.Linux.FrameBuffer` and type the following:
 
 ```dotnetcli
 dotnet run
 ```
 
 The app will start and display on the first available framebuffer device. To change the active framebuffer, set the device name in the `FRAMEBUFFER` environment variable.
 
-`dotnet run` uses the `Debug` configuration, which will show logging information in the current terminal and may overwrite the UI content.
+By default, the `Debug` configuration, which will show logging information in the current terminal and may overwrite the UI content.
```

Comment:
```suggestion
By default, the `Debug` configuration, will show logging information in the current terminal and may overwrite the UI content.
```

Reply:
I think it could be clearer this way?

```suggestion
By default, the `Debug` configuration is used, which will show logging information in the current terminal and may overwrite the UI content.
```